### ISSUE TITLE
History stuck loading, should allow to select entity first

### ISSUE THREAD

**Issue by jtaseff** [home-assistant/frontend#10320]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

The History tab is stuck loading, sometime very slowly, sometimes sits and spins the loading wheel forever. I suspect it's trying to load all of the entity history for the default time window. If it does load, then I'm able to filter by a specific entity and it responds well. But deselecting the entity goes back to loading forever.

### Describe the behavior you expected

I expect to go into history and immediately select a specific entity to view, without having to wait for all the data for all entities to load or for it to get stuck.

### Steps to reproduce the issue

1. Have lots of entities
2. Open the history tab

### What version of Home Assistant Core has the issue?

core-2021.10.4 docker

### What was the last working version of Home Assistant Core?

not sure if HA version or addition of entities

### In which browser are you experiencing the issue with?

Firefox 93.0, Chrome 94.0.4606.81

### Which operating system are you using to run this browser?

Windows 10 21H1

**Comment by spacegaier** [comment]
Duplicate of this feature request https://github.com/home-assistant/frontend/discussions/10174.

Also there is currently an open PR that will add the option to select multiple entities. Once that is in place, we could potentially change the default loading behavior https://github.com/home-assistant/frontend/pull/9946.

**Comment by FrnchFrgg** [comment]
I did not find any bug asking for history to lazy-load visible histories (and a bit more) instead of everything. Maybe that could be worth it to file (or make this bug about that) ? The problem is IMHO orthogonal to entity selection (and starting with empty history feels like a work-around instead).
